### ISSUE TITLE
Bootstrap Select does not initialize when Modal is shown through .Show method

### ISSUE THREAD

**Issue by Mashiane** [creativetimofficial/ct-now-ui-kit-pro#46]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [ X] I am running the latest version
- [ X] I checked the documentation and found no answer
- [ X] I checked to make sure that this issue has not already been filed
- [ X] I'm reporting the issue to the correct repository (for multi-repository projects)

# Expected Behavior

The bootstrap select picker to show on modal

# Current Behavior

The bootstrap select does not show on modal when modal is called with .Show method

# Failure Information (for bugs)

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Define a modal and add a select
2. Call the modal .Show method
3. See attached screenshot as example. I have run the $('select').selectpicker(); call after the modal .Show method and it works.

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Device: Hp
* Operating System: Windows 10
* Browser and Version: Opera 60
![Select](https://user-images.githubusercontent.com/5799306/57201476-afb40200-6f99-11e9-87e0-7895fbf88fc9.png)

## Failure Logs

Please include any relevant log snippets or files here.

**Comment by dragosct10** [comment]
Hi, @Mashiane! Thank you for using our products. Please, tell us if you have any errors in the console? 

Regards, 
Dragos

**Comment by Mashiane** [comment]
Nope, I have run $('#item').selectpicker(); after showing the modal and its now working properly.

**Comment by dragosct10** [comment]
Hi! Please, can you give us your link to the project to see what is happening? 

Regards, 
Dragos
